challenges/mapped-object-types/tests.ts

```diff
@@ -10,52 +12,53 @@ type MoviesByGenre = {
   thriller: 'The Silence of the Lambs';
 };
 
-type Example = MovieInfoByGenre<MoviesByGenre>;
-
-const example: Example = {
```

Comment:
why remove this in favor of `Expect<Equal<`?

Reply:
are these functionally equivalent?

Reply:
well as far as I'm aware they're functionally equivalent, in which case we should revert to the original code (since it's much more "real world"

Reply:
turns out this change broke the example (see https://github.com/typehero/typehero/issues/1019) and was fixed in https://github.com/typehero/typehero/pull/1009